src/WCSimSteppingAction.cc

```diff
@@ -100,7 +100,7 @@ G4int WCSimSteppingAction::G4ThreeVectorToWireTime(G4ThreeVector *pos3d,
 } 
 
 
-void WCSimSteppingAction::Distortion(G4double x,G4double y)
+void WCSimSteppingAction::Distortion(G4double /*x*/,G4double /*y*/)
```

Comment:
Ah, just realizing this wasn't resolved. Let's do these changes to avoid compiler warnings in a separate PR. You could either go through and do them manually after this PR gets merged (kind of annoying, but less of a chance to screw things up), or there is a cherrypick command in git (https://ariejan.net/2010/06/10/cherry-picking-specific-commits-from-another-branch/) which should do what you want but I would recommend using caution (and maybe making a copy of your feature branch in case things go sideways).